lib/v2/datasource/remote/model/member_model.dart

```diff
@@ -0,0 +1,26 @@
+class MemberModel {
+  final String account;
+  final String nickname;
+  final String image;
+
+  MemberModel({this.account, this.nickname, this.image});
+
+  factory MemberModel.fromJson(Map<String, dynamic> json) {
+    return MemberModel(
+      account: json['account'],
+      nickname: json['nickname'],
+      image: json['image'],
+    );
+  }
+
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+          other is MemberModel &&
+              account == other.account &&
+              nickname == other.nickname &&
+              image == other.image;
+
+  @override
+  int get hashCode => super.hashCode;
```

Comment:
All good, just one thing here. 
Do we need to override the inequality operator every time like this? if these classes need to compare instances of the same type for that there is the _Equatable_ lib to save the boilerplate.

```
import 'package:equatable/equatable.dart';

class Person extends Equatable {
  const Person(this.name);

  final String name;

  @override
  List<Object> get props => [name];
}
```
In fact I notice that many models that we went to V2 are doing the same override.

Reply:
> All good, just one thing here.
> Do we need these 2 overrides the equality operator and hashCode every time like this? if these classes need to compare instances of the same type for that there is the _Equatable_ lib to save the boilerplate.
> 
> ```
> import 'package:equatable/equatable.dart';
> 
> class Person extends Equatable {
>   const Person(this.name);
> 
>   final String name;
> 
>   @override
>   List<Object> get props => [name];
> }
> ```
> 
> In fact I notice that many models that we went to V2 are doing the same overrides.

Good point, fixed